Source/Dafny/Dafny.atg

```diff
@@ -2908,8 +2919,9 @@ QSep = "::" | '\u2022'.
  * cardinality brackets, that is, "|expr|".
  */
 Expression<out Expression e, bool allowSemi, bool allowLambda, bool allowBitwiseOps = true>
-= (. Expression e0; IToken endTok; .)
-  EquivExpression<out e, allowSemi, allowLambda, allowBitwiseOps>
+= (. Expression e0; IToken endTok;
+     e = null; .)
+  ( EquivExpression<out e, allowSemi, allowLambda, allowBitwiseOps>
```

Comment:
Is this extra pair of parentheses really needed? If so, indent the lines in between. If not, unindent L2924 to where the `(` is now.